pkg/agent/baker.go

```diff
@@ -322,6 +324,7 @@ func validateAndSetWindowsNodeBootstrappingConfiguration(config *datamodel.NodeB
 // getContainerServiceFuncMap returns all functions used in template generation.
 /* These funcs are a thin wrapper for template generation operations,
 all business logic is implemented in the underlying func. */
+//nolint:funlen,gocognit,nolintlint
```

Comment:
Added nolintlint here because there would be an "unused" error for nolint:funlen without it. Which I think is a bug on golangci side since it is very much used. There is an option to add - unused:true inside the golangci.yaml but I don't think we should totally suppress it. 

On line 140 - https://gitdab.com/Russtopia/xs/commit/5c826f7a5f1791e82f3e98e126bb0a2faa6cd130 - this repository just suppresses the specific message. They also think this is due to a bug in golangci-lint